### ISSUE TITLE
New FireStick - Last 20 seconds to 40 seconds of each vid cut off, jump cuts in movies

### ISSUE THREAD

**Issue by blam666** [koying/SPMC#1013]
Having issues on a new FireStick, bought last week. Running latest regular SPMC.

I'm primarily trying to stream DVD Folders from harddrive. Video streaming works well with Kodi, but I couldn't overcome many sound problems on Kodi (and its "shitty" version).

On SPMC, I don't seem to have any sound problems and DTS option, but the playback is bugged in some respects. The last segments of every video gets cut off too early. It's especially noticable on the trailers before the main movie on my DVDs. Sometimes even the background music in menus is cut off short before the end.

I'm especially interested in keeping DVD structure and trailers intact while streaming, it's the reason I'm using these programs.

Additionally, SPMC seems to sometimes make sudden jump cuts within certain movies and cuts off half a minute or so. But this might be an unrelated issue and so far doesn't seem to happen every time.

If you need any debug logs I'm happy to provide them. I'm really in despair right now, as I tested tons of of different versions of Kodi and now SPMC and really had a different kind of non-fixable problem hell with every version. I'm really hoping SPMC could finally make me happy, if this issue can be overcome.

Any ideas how I could fix this?

**Comment by blam666** [comment]
For some reason, the debug log addon gives an error message if I try to upload the log.
But I'll continue trying this.

I just turned on the timeline while watching a sequence of cut off trailers and it does show the whole running time (like 2 minutes 11 minutes), the grey line is good ahead of the blue line when it happens. Blue Line never gets to its border on the right. Actually it's about a very noticable 10-20 seconds or so cut, not 40 seconds as the title said, sorry.

All the files I'm testing don't cut the vids off this way on Kodi (from 16 to 17.4 to shitty builds.

**Comment by blam666** [comment]
Sorry, I DID check other posts before, but not for the right keywords, it seems. By now, I discovered this is the same problem as in the DVD ISO issue that was discussed in #841. On the older SPMC version 16.5.5 everything works fine. Sorry!

Now I have to look for a way to get the TMDB scraper working again in that version. The new dev version I tried did have problems with accessing my video folders by SMB (gave error message). I have to look further into this.
